doc/doxygen/Doxylocal

```diff
@@ -1,4 +1,3 @@
 INPUT                  += @abs_top_srcdir@/doc/doxygen/mainpage \
                          @abs_top_srcdir@/doc/doxygen/modules \
-                         @abs_top_srcdir@/dune/common/ \
-                         @abs_top_srcdir@/dune/grid/
+                         @abs_top_srcdir@/opm/grid /
```

Comment:
What does this&ndash;i.e., a single trailing `/` character&ndash;do to the configuration?  Does it mean we effectively search the full tree rooted at `/` or does it do something else (or nothing at all)?